### DISCUSSION TITLE
Phase delay plugin

### DISCUSSION THREAD

**Discussion by Tine1994** [wwmm/easyeffects#1395] [Ideas]
Hi, community :)

Do you think it is possible to implement a Plugin to get a setting for Phase delay in the Audio path?

When people are using full range Speakers in small rooms, a phase delay setting is very useful to delete standing waves or bass walking into other rooms.

Would be great to get such a setting :)

**Comment by wwmm** [comment]
>Do you think it is possible to implement a Plugin to get a setting for Phase delay in the Audio path?

At this moment I do not have the knowledge for that. So we will need to use someone else's plugins. If you find a LV2 plugin or library that does what you need let me know. that being said one of the plugins already available in EasyEffects seems to have some kind of phase control
![Screenshot from 2022-02-11 11-12-16](https://user-images.githubusercontent.com/16225172/153607132-73307470-75b5-4ca1-a0cd-7510b49990c6.png)
The `Stereo Tools` plugin is from the `Calf Gears Studio`. I have never tried to see what the phase control is doing exactly. As the name suggests it should be changing the signal phase. But if it is in the same way for both channels I am not sure.

**Reply by wwmm** [reply]
The plugin's manual does not say much about this setting https://github.com/calf-studio-gear/calf/blob/master/doc/manuals/Stereo%20Tools.html. It may not be the same as adding a delay to the phase.

**Reply by Tine1994** [reply]
That is the right feature, i am seaching. I measured at the speakers output and its exactly the phase delay i mean. Between left and right speaker. Thats perfect!
How can we share that that also other people know that "phase delay" is available in PulseEffects? 
Thats the greatest Tool ever!

**Reply by wwmm** [reply]
>How can we share that that also other people know that "phase delay" is available in PulseEffects?

I don't know. We do not have any channel in a website that could be used for that. But based on what I see on google from time to time people seem to post about EasyEffects(PulseEffects) on `Reddit` in cases like this.
